zoomus/__init__.py

```diff
@@ -7,4 +7,4 @@
 
 
 __all__ = ["API_VERSION_1", "API_VERSION_2", "ZoomClient"]
-__version__ = "1.1.3"
+__version__ = "1.1.4"
```

Comment:
Please revert this change and leave it as 1.1.3 -- I'll update it to the appropriate version when I cut the next release (which likely will be 1.1.6).

Reply:
Sure Did it